doc/rtd/topics/cli.rst

```diff
@@ -215,35 +230,57 @@ aliases:
     v1
     vendordata
 
-Below demonstrates how to query standardized metadata from clouds:
+Here are a few examples of how to query standardized metadata from clouds:
+
+.. code-block:: shell-session
+
+   $ cloud-init query v1.cloud_name
+
+Example output:
+
+.. code-block::
```

Comment:
I'm not 100% sure exactly what the output for this one should look like. Is it another one of those cases where the comment isn't part of the output and should be moved outside the code block? If so I might be able to rework the text so that we don't even need the code block.

Reply:
Nevermind, there was a space missing between the .. and code-block...d'oh!